python/snewpy/models/ccsn.py

```diff
@@ -742,6 +742,51 @@ def __new__(cls, filename=None, *, progenitor_mass=None):
     # Populate Docstring with abbreviated param values
     __new__.__doc__ = __new__.__doc__.format(**_param_abbrv)
 
+
+class Mori_2023(_RegistryModel):
+    """Model based on 2D simulations with axionlike particles, K. Mori, T.  Takiwaki, K. Kotake and S. Horiuchi, Phys. Rev. D 108:063027, 2023. All models are based on the non-rotating 20 M_sun solar metallicity progenitor model from S.E. Woolsey and A. Heger, Phys. Rep. 442:269, 2007. Data from private communication.
+        """
+    param = {'axion_mass' : [0, 100, 200],
+             'axion_coupling' : [0, 2, 4, 6, 8, 10, 12, 14, 16, 20] }
+
+    _param_validator = lambda p: \
+        (p['axion_mass'] == 0 and p['axion_coupling'] == 0) or \
+        (p['axion_mass'] == 100 and p['axion_coupling'] in (2,4,10,12,14,16,20)) or \
+        (p['axion_mass'] == 200 and p['axion_coupling'] in (2,4,6,8,10,20))
+
+    _param_abbrv = {
+        'axion_mass': '[0, 100, 200] MeV',
+        'axion_coupling' : '[0..20] 1e-10/GeV' }
+
+    def __new__(cls, axion_mass, axion_coupling):
+        """Model Initialization.
+
+        Parameters
+        ----------
+        axion_mass: int
+            Axion mass in units of MeV. Valid values are {axion_mass}.
+        axion_coupling: int
+            Axion-photon coupling, in units of 1e-10/GeV. Valid values are {axion_coupling}.
+        """
+        # Load from Parameters
+        cls.check_valid_params(cls, axion_mass=axion_mass, axion_coupling=axion_coupling)
+
+        if axion_mass == 0:
+            filename = 't-prof_std.dat'
+        else:
+            filename = f't-prof_{axion_mass}_{axion_coupling}.dat'
+
+        metadata = {
+            'Axion mass': axion_mass * u.MeV,
+            'Axion coupling': axion_coupling * 1e-10/u.GeV,
```

Comment:
And remove the multiplying by units here
```suggestion
            'Axion mass': axion_mass,
            'Axion coupling': axion_coupling,
```